validation/superchain-genesis_test.go

```diff
@@ -16,6 +16,11 @@ import (
 
 func testGenesisHash(t *testing.T, chainID uint64) {
 	skipIfExcluded(t, chainID)
+
+	if chainID == 10 {
+		t.Skip("OP Mainnet has a pre-bedrock genesis (with an empty allocs object stored in the registry), so we exclude it from this check.")
+	}
```

Comment:
Why is this exclusion called out separately whereas the other exclusions are defined in the exclusions_test.go file?

Reply:
This is because it is a really unique exceptional case, and the new `TestExclusions` test requires that no standard chain is excluded from any check. I tidied this up a bit [4c73866](https://github.com/ethereum-optimism/superchain-registry/pull/385/commits/4c73866516003dd9fd1da6ee9dabd96a064754e7).